src/h3lib/include/h3api.h.in

```diff
@@ -292,79 +298,79 @@ double H3_EXPORT(cellAreaKm2)(H3Index h);
 double H3_EXPORT(cellAreaM2)(H3Index h);
 /** @} */
 
-/** @defgroup edgeLength edgeLength
- * Functions for edgeLength
+/** @defgroup getHexagonEdgeLengthAvg getHexagonEdgeLengthAvg
+ * Functions for getHexagonEdgeLengthAvg
  * @{
  */
 /** @brief average hexagon edge length in kilometers (excludes pentagons) */
-double H3_EXPORT(edgeLengthKm)(int res);
+double H3_EXPORT(getHexagonEdgeLengthAvgKm)(int res);
 
 /** @brief average hexagon edge length in meters (excludes pentagons) */
-double H3_EXPORT(edgeLengthM)(int res);
+double H3_EXPORT(getHexagonEdgeLengthAvgM)(int res);
 /** @} */
 
 /** @defgroup exactEdgeLength exactEdgeLength
  * Functions for exactEdgeLength
  * @{
  */
-/** @brief exact length for a specific unidirectional edge in radians*/
+/** @brief exact length for a specific directed edge in radians*/
 double H3_EXPORT(exactEdgeLengthRads)(H3Index edge);
 
-/** @brief exact length for a specific unidirectional edge in kilometers*/
+/** @brief exact length for a specific directed edge in kilometers*/
 double H3_EXPORT(exactEdgeLengthKm)(H3Index edge);
 
-/** @brief exact length for a specific unidirectional edge in meters*/
+/** @brief exact length for a specific directed edge in meters*/
 double H3_EXPORT(exactEdgeLengthM)(H3Index edge);
 /** @} */
```

Comment:
These should just be `edgeLengthRads`, `edgeLengthKm`, and `edgeLengthM`, since we can drop the `exact` because we no longer have a naming conflict with `getEdgeLengthAvg`.

Reply:
@dfellis bump :)

Reply:
Actually, never mind. I see that these aren't in the RFC. I'll make an issue for a follow-up PR to change these.

Reply:
https://github.com/uber/h3/issues/442